gammapy/spectrum/tests/test_spectrum_fit.py

```diff
@@ -27,9 +27,9 @@ def test_spectral_fit():
     fit.run()
 
     assert fit.result[0].fit.spectral_model == 'PowerLaw'
-    assert_allclose(fit.result[0].fit.statval, 105.398, rtol=1e-3)
+    assert_allclose(fit.result[0].fit.statval, 103.595, rtol=1e-3)
```

Comment:
Changes in FitStatistic and Index are due to 71 -> 72 Reco energy bins (72 yields 12 bpd, the value before was 'wrong').